src/client/pages/Payment/Breadcrumbs.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { useTextKeys } from 'utils/textKeys'
+
+export const Breadcrumbs = () => {
+  const textKeys = useTextKeys()
+  return (
+    <Root>
+      <Step>{textKeys.BREADCRUMB_CHECKOUT()}</Step>
+      <Step active>{textKeys.BREADCRUMB_PAYMENT()}</Step>
```

Comment:
Don't we need to control which one is active? Part of another PR maybe?

Reply:
The only option is the payment step -- we navigate back to racoon and a different page after success :)